paper/paper.md

```diff
@@ -0,0 +1,281 @@
+bibliography: paper.bib
+
+# Optional fields if submitting to a AAS journal too, see this blog post:
+# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
+# aas-doi: 10.3847/xxxxx <- update this with the DOI from AAS once you know it.
+# aas-journal: Astrophysical Journal <- The name of the AAS journal.
+---
+
+# Statement of Need
+
+The field of Magnetic Resonance Imaging (MRI) Connectomics has rapidly expanded since its advent 
+  in the 2000s [@SpornsTheBrain:2005], [@HagmannFromConnectomics:2005], [@SpornsEditorialConnectomics:2018].
+It has revolutionized the way to investigate `in-vivo`, `non-invasively` and 
+  `safely` at different macroscopic scales the structural and functional systems of the 
+  brain by modeling connections between brain areas as a graph or network, the so-called
+  `connectome`, and has become a widely used method in Neuroscience [@Bassett2017]. 
+While brain areas are usually derived from high resolution structural T1 weighted MRI (sMRI),
+  structural brain connectomes are mapped from diffusion MR imaging (dMRI) tractography, and
+  functional brain connectomes are usually mapped from resting-state functional MRI (rfMRI).
+As MRI is being increasingly more accessible and used in both clinical and research settings,
+  such multi-modal MRI datasets are being gathered at an unprecedented rate.
+The size and organization of these datasets as well as the increasing complexity of the processing
+  pipelines to analyze them present important challenges for scalable computing, data sharing,
+  reproducibility and transparency of the analysis.
+
+The last ten years have indeed witnessed a number of technical advances and a trend towards the adoption
+  by the community of open research practices, which promotes `open data` and `open method`,
+  to address these challenges [@Nichols:2017] [@EglenTowardNeuroscience:2017] [@Kennedy2019].
+This has led in particular to the creation of a community standard, known as the Brain Imaging Data Structure (BIDS),
+  initially designed to ease the practice of sharing raw MRI data [@GorgolewskiTheExperiments:2016]. 
+[@GorgolewskiBIDSMethods:2017].
+Combined with advances in software virtualization, BIDS has enabled the creation of the BIDS Apps
+  framework which uses software container technology to encapsulate neuroimaging processing pipelines
+  and ensure portability and reproducibility.
+A large ecosystem of processing pipelines supporting the mapping of connectomes has evolved around this framework,
+  including C-PAC [@cpac:2013], NIAK [@BellecNeuroimagingNIAK:2016], fMRIPrep [@Ghosh:2018], dMRIPrep [@dmriprep:2019],
+  QSIPREP [@CieslakQSIPrep:2020], MRtrix3_connectome [@Smith2019:BIDSApp], NDMG [@KiarAVariability:2018]
+  and PyNets [@PisnerPyNets:2020], which have all demonstrated their capability in addressing all the previously-mentioned
+  challenges.
+However, none of the existing solutions provide a direct alternative to Connectome Mapper.
```

Comment:
Shameless self-citation ;) - Consider adding: 'However, none of the existing solutions provide a direct alternative to Connectome Mapper when dealing with multimodal datasets with the goal to create connectomes.'

Reply:
see also direct competitor to CMP:
https://www.biorxiv.org/content/10.1101/2022.01.31.478189v1

Reply:
Thanks for the link @jwirsich ! I will add it to the list. 
```suggestion
However, none of the existing solutions provide a direct alternative to Connectome Mapper
  when dealing with multimodal datasets with the goal to create connectomes at multiple
  scales with hierarchical region grouping.
```